mx.pwmc.Svgvi.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "mx.pwmc.Svgvi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "writable-sdk": true,
```

Comment:
This shouldn't be needed. What error did you fix with it?

Reply:
This is need by dependencies:
` "writable-sdk": true `

They add GObject Introspection GIR/TYPELIB and without this setting it is not possible and fails.

Reply:
Sounds like it just installs files at wrong location. Which dependencies do so?

```
"build-options": {
        "env": {
            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
        }
    },
```

is usually enough to make it work.